R/Angel Chen/ticket #20492 Norman.R

```diff
@@ -0,0 +1,144 @@
+#ticket #20492 Norman
+#Human dataset:
+#https://arcticdata.io/catalog/#view/urn:uuid:c715fe12-6c05-4c27-8d00-b9c0c536c54b
+
+pkg <- get_package(adc, 'resource_map_urn:uuid:2a957ad3-1c0f-44e9-b79d-5ea2a2ed76bf', file_names = TRUE)
+doc <- read_eml(getObject(adc, pkg$metadata))
+emld::eml_version("eml-2.1.1")
+
+id_new <- publish_object(
+  adc,
+  path = "~/ticket #20492 Norman/CEBP_HumanGeneticData_Summary.csv",
+  format_id = "text/csv",
+  public = FALSE
+)
+
+update <- publish_update(adc, 
+                         metadata_pid = pkg$metadata,
+                         resource_map_pid = pkg$resource_map,
+                         data_pids = id_new,
+                         public = FALSE)
+#https://arcticdata.io/catalog/view/urn%3Auuid%3Ab7b1a184-4f25-4c7f-b66d-240e2dfcd348
+
+pkg <- get_package(adc, 'resource_map_urn:uuid:b7b1a184-4f25-4c7f-b66d-240e2dfcd348', file_names = TRUE)
+doc <- read_eml(getObject(adc, pkg$metadata))
+emld::eml_version("eml-2.1.1")
+
+doc <- eml_add_publisher(doc)
+doc <- eml_add_entity_system(doc)
+
+CEBP_HumanGeneticData_Summary <- read_csv("~/ticket #20492 Norman/CEBP_HumanGeneticData_Summary.csv")
+out <- shiny_attributes(CEBP_HumanGeneticData_Summary, NULL)
```

Comment:
I'm curious what this step was used for, could you explain?

Reply:
@angelchen7

Reply:
@eeerika 
Oops, sorry I didn't see your message until now! If I have to make attributes list from scratch and if there are a lot of attribute names, I read in the csv file and plug that as an argument to shiny_attributes() so that the attribute names are already there for me in the shiny app when I open it! I hope that explains it well enough....